src/locale/locales/ja/messages.po

```diff
@@ -1445,7 +1445,7 @@ msgstr "{0}のみ返信可能"
 #: src/view/com/composer/Composer.tsx:468
 #: src/view/com/composer/Composer.tsx:469
 msgid "Open emoji picker"
-msgstr ""
+msgstr "絵文字ピッカーを開く"
```

Comment:
「絵文字ピッカー」はITに詳しくないと伝わらない用語という気がするので、「絵文字を入力」などの方が一般向けとして相応しいように感じました。

I feel that "絵文字ピッカー" is a term that only those who are familiar with IT would understand, so I think that something like "絵文字を入力" would be more appropriate for the general public.

Reply:
Has this change been made?